src/mk-s390-cdboot

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python3
```

Comment:
Are there any other tools that do this? LGTM if there aren't, but if there are we should look into using those.

Reply:
Or we can add `lorax` RPM for `deps-s390x.txt`, so we don't have to copy this file here ? I guess `mk-s390-cdboot` binary should be in that RPM.
As of now, I only need this standalone program from `lorax`.

Reply:
`/usr/bin/mk-s390-cdboot` seems to be provided by lorax. I think it's cleaner if we add `lorax` RPM. Good idea, thanks. Updating.